### ISSUE TITLE
Would you mind to cut a release?

### ISSUE THREAD

**Issue by hannesm** [dbuenzli/cmarkit#13]
Dear Daniel,

I'm very happy with this library -- thanks a lot for writing it and publishing it. Unfortunately I just ran into #10 (or actually something else, but related enough that fixing 10 fixed it) -- I use some tables with empty cells, and the following table is rendered inappropriately with your latest release (0.2):

```markdown
| a           | | b         |  c            | d | e |
| --------------------- |-| -------------- | -------------- | --------------- | ------ |
| entry         | | | **bla.file** | **other.file**     |                 |
| _another entry_  | | | | **another.file** | **another.other** |
```

The HTML rendering of cmarkit is:
```html
<div role="region"><table>
<tr>
<th>a</th>
<th>b</th>
<th>c</th>
<th>d</th>
<th>e</th>
<th></th>
</tr>
<tr>
<td>entry</td>
<td><strong>bla.file</strong></td>
<td><strong>other.file</strong></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td><em>another entry</em></td>
<td><strong>another.file</strong></td>
<td><strong>another.other</strong></td>
<td></td>
<td></td>
<td></td>
</tr>
</table></div>
```

Thus, the empty cells are pushed to the end of the rows.

With the latest commit on the main branch (fd21e5f), the rendering looks nicely:
```html
<div role="region"><table>
<tr>
<th>a</th>
<th></th>
<th>b</th>
<th>c</th>
<th>d</th>
<th>e</th>
</tr>
<tr>
<td>entry</td>
<td></td>
<td></td>
<td><strong>bla.file</strong></td>
<td><strong>other.file</strong></td>
<td></td>
</tr>
<tr>
<td><em>another entry</em></td>
<td></td>
<td></td>
<td></td>
<td><strong>another.file</strong></td>
</tr>
 </table></div><section role="doc-endnotes"><ol>
```

As reference, here's the GitHub rendered table:

| a           | | b         |  c            | d | e |
| --------------------- |-| -------------- | -------------- | --------------- | ------ |
| entry         | | | **bla.file** | **other.file**     |                 |
| _another entry_  | | | | **another.file** | **another.other** |

TL;DR: I'd appreciate you pushing the release button. Thanks again a lot for cmarkit.

**Comment by dbuenzli** [comment]
I can do that. But I should have a look at solving https://github.com/dbuenzli/cmarkit/issues/11 before.

**Comment by dbuenzli** [comment]
https://github.com/ocaml/opam-repository/pull/24931

**Comment by hannesm** [comment]
Thanks a lot!
